### PR TITLE
Remove loop in assemble

### PR DESCRIPTION
**Pull request by fredrikekre**
Some artifact from calfem perhaps?

### CODE REVIEW

**Comment by fredrikekre** [comment]
Btw, should we make `assemble` -> `assemble!` ?

**Comment by fredrikekre** [comment]
I think it would be better "flow" if the order of the input was

```jl
assemble(a::Assembler, Ke::Matrix, edof::Vector)
```

i.e. "assemble into `a`, the matrix `Ke`, on position `edof`".

File: src/assembler.jl [hunk 1]
```diff
@@ -1,7 +1,7 @@
-type Assembler
+type Assembler{T}
```
Comment (fredrikekre) [inline src/assembler.jl]: `immutable`?

Comment (KristofferC) [inline src/assembler.jl]: Yes.

**Comment by KristofferC** [comment]
Order was a left over from calfem. Makes sense to change and call it `assemble!`

**Comment by fredrikekre** [comment]
Oh look! The review is outdated, nice.

**Pull request merged**